### DISCUSSION TITLE
Suggestion for security vulnerabilities

### DISCUSSION THREAD

**Discussion by 98-jeonghoon** [sherlock-project/sherlock#1196] [Ideas]
What do you think about creating a SECURITY.md file so that we can proceed with bug bounties for security vulnerabilities like other open source do?

For other open source vulnerabilities, the SECURITY.md file is managed with the link below.
https://github.com/tiangolo/fastapi/blob/master/SECURITY.md
https://github.com/scikit-learn/scikit-learn/blob/main/SECURITY.md
https://github.com/pytorch/pytorch/blob/master/SECURITY.md

**Comment by sdushantha** [comment]
That seems like a nice idea!

> so that we can proceed with bug bounties for security vulnerabilities

But we unfortunately wont be giving out bounties for the vulnerabilities that are found
